### DISCUSSION TITLE
Add collapsed state to relational interfaces

### DISCUSSION THREAD

**Discussion by rijkvanzanten** [directus/directus#5198] [Feature Requests]
The interface can grow pretty large if you have a lot of related items. We should add a way to only show the first X and have a button to show all of them

**Comment by jaads** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
